### DISCUSSION TITLE
Notes

### DISCUSSION THREAD

**Discussion by bunchy7ss** [ente-io/ente#2883] [Enhancements]
Alot of my accounts have something special to them, or another form of text based security,
i would like a way to add a note to accounts. like for example
Google
mygoogleemail@gmail.com
404 404
NOTE: this account is used for spam!

**Comment by haritvsmurali** [comment]
must have feature! but until this feature is implemented you can use ACCOUNT field to update own comments.

**Comment by hrshtein** [comment]
I also need this feature to add recovery codes.

**Reply by homeostashish** [reply]
i think its preferable to add recovery codes to your existing password manager..

**Reply by hrshtein** [reply]
I am still storing my seed, recovery codes etc in my password manager, but i want to switch because i dont want to keep all my eggs in one!

Is there any other way you recommend to store this?

**Reply by homeostashish** [reply]
These are meant to be stored in a password manager, I suppose you are using the phrase Incorrectly. It's like saying I'll store my social media passwords in a separate app. Pass manager and totp should be kept in "different baskets". But if you think otherwise, use a different password manager for you backup codes. Totally depends on your use case.

**Comment by brog-io** [comment]
This has been implemented in the latest version

**Comment by bunchy7ss** [comment]
Yay
